controllers/cre/controller.go

```diff
@@ -137,6 +137,7 @@ func (r *CustomRuntimeEnvironmentReconciler) reconcilePipelineRun(ctx context.Co
 		if errors.IsNotFound(err) {
 			logger.Info("Creating PipelineRun")
 
+			// TODO: replace ArrayOrString (deprecated) https://github.com/tektoncd/pipeline/blob/75d0037daf06664cb93efe86d974e1fbf69cbbbe/pkg/apis/pipeline/v1beta1/param_types.go#L140
```

Comment:
@KPostOffice could you relate the tekton upstream version to an OpenShift product version?

Reply:
The comment exists from 0.39.0 onward. I found [this](https://docs.openshift.com/container-platform/4.11/cicd/pipelines/op-release-notes.html#compatibility-support-matrix_op-release-notes) compatibility matrix. It states that operator version 1.8, which is the latest pipeline operator version, is tekton version 0.37.x.

The version we are currently using in this repo is v0.38.3. This isn't an issue for now.